app/services/laa_reference_unlinker.rb

```diff
@@ -24,6 +24,7 @@ def unlink_maat_reference!
   def push_to_queue
     Sqs::MessagePublisher.call(
       message: {
+        defendantId: defendant_id,
         maatId: laa_reference.maat_reference,
```

Comment:
So we're still sending MAAT ID?

Reply:
two-prong approach:

1. add defendantId and if everything is fine
2. remove maatId